endoreg_db/views/pdf/pdf_stream.py

```
@@ -162,33 +156,31 @@ def get(self, request, pk: int, *args, **kwargs):
             try:
                 with open(file_path, "rb") as file_handle:
                     response = FileResponse(file_handle, content_type="application/pdf")
                     response["Content-Length"] = str(file_size)
                     response["Accept-Ranges"] = "bytes"
-                    response["Content-Disposition"] = (
-                        f'inline; filename="{safe_filename}"'
-                    )
+                    response["Content-Disposition"] = f'inline; filename="{safe_filename}"'
 
                 # FileResponse will take ownership of file_handle and close it after response
                 return response
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)